lib/karafka/schemas/consumer_group.rb

```diff
@@ -3,7 +3,9 @@
 module Karafka
   module Schemas
     # Schema for single full route (consumer group + topics) validation.
-    ConsumerGroup = Dry::Validation.Schema do
+    class ConsumerGroup < Dry::Validation::Contract
+      config.messages.load_paths << File.join(Karafka .gem_root, 'config', 'errors.yml')
```

Comment:
Is there supposed to be a space here `Karafka .gem_root`? Noticed ResponderUsageTopic also has the same syntax.

Reply:
Good catch. Checking why the CI didn't catch that...

Reply:
Dunno. Not catched. Fixed. Thanks!

Reply:
Apparently it’s actually valid syntax 🤷‍♂️

Reply:
Yeah. Weird. I believe RuboCop should catch that and Coditsu should fail on Circle. Didn't but fixed. Thank you :heart: